domains/mateoguidi.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Mateo GUIDI's Portfolio",
+    "repo": "https://github.com/MateoGuidi/portfolio",
+    "owner": {
+        "username": "MateoGuidi",
+        "email": "mateo.guidi5@gmail.com"
+    },
+    "record": {
+        "CNAME": "mateoguidi.github.io/portfolio"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "mateoguidi.github.io"
```